applications/dashboard/models/class.usermodel.php

```diff
@@ -836,9 +841,14 @@ public function connect($uniqueID, $providerKey, $userData, $options = []) {
         trace('UserModel->Connect()');
         $provider = Gdn_AuthenticationProviderModel::getProviderByKey($providerKey);
 
+        $isTrustedProvider = isset($provider['Trusted']) ? $provider['Trusted'] : false;
```

Comment:
This would be a perfect fit for the [null coalescing operator](http://php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op). I think there are a few other candidates for null-coalescing in here.

If the "Trusted" field of `$provider` isn't already a boolean, I would cast it to one when you're defining `$isTrustedProvider`.